lib/nrf_modem_lib/Kconfig

```diff
@@ -69,6 +69,12 @@ endif # NRF_MODEM_LIB_TRACE_MEDIUM_RTT
 
 endif # NRF_MODEM_LIB_TRACE_ENABLED
 
+config NRF_MODEM_LIB_AT_CMD_FILTER
+	bool "Enable AT command filters"
```

Comment:
As per the guidelines, please remove `Enable`:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.